tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java

```diff
@@ -456,11 +460,35 @@ private boolean shouldShowMessageDialog( IProject project )
         return markers != null && markers.length > 0;
     }
 
+    @SuppressWarnings( "deprecation" )
     private boolean shouldAdd( Problem problem )
     {
-        String path = problem.getFile().getAbsolutePath().replaceAll( "\\\\", "/" );
+        File file = problem.getFile();
```

Comment:
Lets change this code to use the new 'find a target' code that you changed in another pull request.